### ISSUE TITLE
Failed to load project file: Ran out of input

### ISSUE THREAD

**Issue by tiabutterfly** [OpenShot/openshot-qt#1621]
TL;DR, last night I had some issues with OpenShot, today I go to open the project I was working on and get an error message. Can somebody help me recover my project?

Operating system: Windows 7 Professional, 64-bit.
OpenShot version 2.4.1.

The error message:
Error Opening Project: Failed to load project file C:/Users/myname/Desktop/foldername/filename.osp: Ran out of input
![image](https://user-images.githubusercontent.com/39709667/40626911-2d27065a-6281-11e8-8248-55e7a34faf4d.png)

The long version of the story:
Last night I was working on a project in OpenShot when the program stopped responding.
The video froze, but the audio kept playing, albeit in a stop-start manner. This is an issue that I've experienced before and usually resolves itself with time; I press the pause button, and within a few minutes the program pauses the audio, and the video catches up before also pausing.
This time, though, the problem persisted.
I pressed the pause button as usual and waited it out. I waited long enough for the entire song I was working with to play out in full, while stopping and starting, and then some. The video continued to be frozen and the program continued being unresponsive.
After twenty minutes or so I finally decided it wasn't going to resolve itself, and so hit ctrl-alt-delete in an attempt to close the program with Task Manager. By this point, I figured it would take less time to redo the work I hadn't yet saved than to continue waiting it out.
Nothing happened for a few minutes after I pressed ctrl-alt-delete. I may have repeated the sequence once or twice.
Then my screen went dark.
Baffled, I moved the mouse around and pressed spacebar a few times. I heard some noises that sounded like Windows error noises, but the screen remained dark. I turned the screen itself off and back on, but it still was dark.
Ultimately I decided to do a hard reboot, since my entire computer was unresponsive at this point. I powered my computer down using the power button, then turned it back on the same way, and restarted Windows normally. By this point it was pretty late, later than I usually stay up. I attempted to reopen OpenShot to check how much work I'd lost, but I was tired and frustrated, so when the program didn't actually load I shut my computer off for the night.
Skip to today. I turn my computer on, sit through the default programs starting up, close them all and open OpenShot. I go to open the project I was working on last night, and get an error.
Error Opening Project: Failed to load project file C:/Users/myname/Desktop/foldername/filename.osp: Ran out of input
I click OK, then try opening the project again. Same error. I close OpenShot, reopen it, open a different file. This one opens fine. Try another different file (that one initially created with a previous version of OpenShot), it also opens fine. Try last night's project again. Same error.
Looking the error up yielded nothing particularly helpful. Is it possible to recover my project? What can I do?

**Comment by DylanC** [comment]
@tiabutterfly - I hate to say it but it sounds like your project file was not fully saved or interrupted during the save process. Can you post the .osp file here? We could look at it and see if its fixable.

**Comment by tiabutterfly** [comment]
@DylanC GitHub is telling me the file type isn't supported.
![image](https://user-images.githubusercontent.com/39709667/40661664-f61f6154-6319-11e8-9237-ffd8f9c0342b.png)

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Anything here, Cap'ain? o.O

**Comment by tiabutterfly** [comment]
Anybody?

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Already gave my opinion on the matter and it seems to be true. The .osp file sent on is blank, so the file was interrupted during the save process. 

@tiabutterfly - You project file is blank perhaps as a result of the save getting interrupted. Sadly I'm not sure there is really much we can do here.

**Comment by peanutbutterandcrackers** [comment]
Ooops. My bad. I didn't even check the `.osp` file. 

@tiabutterfly - Sorry. It seems your `.osp` file is, indeed, blank; and there isn't much we can do about it. Sorry.

**Comment by DylanC** [comment]
@tiabutterfly - We would help you if possible but unfortunately we can't in this situation due to your file simply getting lost during the save process. 

All I can recommend is to start again but definitely use the **daily builds** as you should have much less trouble with this version. If you want you could also wait until v2.4.2 is released which will be almost the same as the current daily builds.
